test/built-ins/Map/newtarget.js

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2015 the V8 project authors. All rights reserved.
```

Comment:
This test appears to be a duplicate of test/built-ins/Map/properties-of-map-instances.js